rootfs/scheduler/__init__.py

```diff
@@ -1094,6 +1098,42 @@ def _buildpack_readiness_probe(self, controller, delay=30, timeout=5, period_sec
 
         return controller
 
+    '''
+    Applies tcp socket readiness probe to the docker app container only if some port is exposed
+    by the docker image.
+    '''
+    def _dockerapp_readiness_probe(self, controller, image, delay=5, timeout=5, period_seconds=5,
```

Comment:
since readiness probe gets executed after the initial delay  from the start of the pod i think we should a bit early as it might start serving immediately unlike the slug type apps where we know for sure that it has to download the slug before it can start serving requests.